dataspace-connector/full/README.md

```diff
@@ -18,15 +18,17 @@ To get and install your own IDS identity certificate, please read the provided [
 
 In this setup, the [Dataspace Connector](https://github.com/International-Data-Spaces-Association/DataspaceConnector) is started with the listed extensions:
 * [Dataspace Connector UI](https://github.com/International-Data-Spaces-Association/DataspaceConnectorUI), 
+* Portainer
 * PostgreSQL DB
 
 ### Components & Versions
 This deployment example builds on the following compatible versions:
 
 | Component | Version |
 |:----------|:--------|
-| Dataspace Connector | 6.1.0 |
-| Dataspace Connector UI | 8.3.2 |
+| Dataspace Connector | 6.1.3 |
```

Comment:
The version of the DSC will change to 6.2.0.

Reply:
I changed the version to 6.2.0

Reply:
needs to be changed in docker-compose too

Reply:
I changed it in the docker-compose to 6.2.0